docs/containers.conf.5.md

```diff
@@ -621,14 +621,22 @@ Number of CPU's a machine is created with.
 
 The size of the disk in GB created when init-ing a podman-machine VM
 
-**image**="testing"
+**memory**=2048
```

Comment:
Why are you changing the sort order.  I think this file is supposed to be sorted per/table.

Reply:
My thinking was to improve readability by grouping the physical parameters together and the image related ones together. I thought about proposing a struct order change as well, but wasn't sure about possible compatibility problems.  I can back the sort change out if you prefer.

Reply:
Yes keep it sorted alphabetically.